### PR TITLE
Further records for Miracles of Jesus and other work records

### PR DESCRIPTION
**Pull request by Ralph-Lee-UK**
@thea-m there are a few additional files here for MJ, and a few others that have been created following discussions on GitHub

### CODE REVIEW

File: new/LIT6451MMMonkRain.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="RL" when="2021-05-14">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography"/><!---->
```
Comment (thea-m) [inline new/LIT6451MMMonkRain.xml:53]: Could you add a relation to LIT2384Taamme?

File: new/LIT6453MalkeaMichaelGabriel.xml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography"/>
+            <div type="edition">
+                <note>Transcribed from EMIP 2723</note>
+                <div type="textpart" subtype="incipit">
+                    <ab>
+                        ሰላም፡ ለዝክረ፡ ሰምክሙ፡ ከመ፡ ወርኀ፡ ዘአሠነያ።
```
Comment (thea-m) [inline new/LIT6453MalkeaMichaelGabriel.xml:59]: ስምክሙ?

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you for all the work! Just some small questions.
It would be extremely helpful in the future if you could link to the issues in which these works were discussed in the PR (I think it's always easiest for the author to do this, because they have the best overview), this really makes a big difference in the long term in searching for old issues and being able to follow what was decided.

**Comment by thea-m** [comment]
https://github.com/BetaMasaheft/Documentation/issues/1743

**Comment by Ralph-Lee-UK** [comment]
@thea-m I realise now that links to the discussions would have been extremely helpful! Noted!

File: new/LIT1528GadlaZ.xml [hunk 3]
```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="work" xml:lang="en" xml:id="LIT1528GadlaZ">
+   <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title xml:lang="gez" xml:id="t1">ገድለ፡ ዘርአ፡ ቡሩክ፡</title>
```
[the remark this answers is not in the document]
Comment (Ralph-Lee-UK) [reply new/LIT1528GadlaZ.xml:7]: Is this instead of 'Vita of Zarʾa Buruk'?

[the remark this answers is not in the document]
Comment (Ralph-Lee-UK) [reply new/LIT1528GadlaZ.xml:7]: I think I have done what you are suggesting - keeping the other titles but making this the first one.

File: new/LIT1528GadlaZ.xml [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+            <listBibl type="translation">
+               <bibl>
+                  <ptr target="bm:Ricci1979ZaraBuruk"/>
+               </bibl>
+            </listBibl>
+            <listBibl type="secondary">
+               <bibl>
+                  <ptr target="bm:Sokolinskaia2014ZaraBuruk"/>
+               </bibl>
+            </listBibl>
```
[the remark this answers is not in the document]
Comment (Ralph-Lee-UK) [reply new/LIT1528GadlaZ.xml:86]: I am a little confused here. The file is LIT1528GadlaZ - is there another file that you are suggesting a relation to?

[the remark this answers is not in the document]
Comment (thea-m) [reply new/LIT1528GadlaZ.xml:86]: Ah, I must have been confused by the green colour and the fast that this record is now of the hagiographic dossier, not only the gadl, sorry. It is very strange that the old files show up as new here. @PietroLiuzzo is this something we should worry about or can we just go ahead normally?

File: new/LIT1528GadlaZ.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (thea-m) [reply new/LIT1528GadlaZ.xml:7]: Thank you - however I can see no changes to any of these files. Are you sure you committed them on the same branch?

[the remark this answers is not in the document]
Comment (Ralph-Lee-UK) [reply new/LIT1528GadlaZ.xml:7]: @thea-m apologies. I had to leave this for some other work, and now returning to it I see that I didn't save the work! I will try to address these things again today and remember to push the changes!

File: new/LIT6445LetusWeep.xml [hunk 5]
```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6445LetusWeep" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ንዑ፡ ንብኪ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Nǝʿu nǝbǝki</title>
```
[the remark this answers is not in the document]
Comment (Ralph-Lee-UK) [reply new/LIT6445LetusWeep.xml:9]: Corrected

File: new/LIT6448HymnTrinity.xml [hunk 6]
```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6448HymnTrinity" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">መላእክቲሆሙ፡ ለሥሉስ፡ ሩጸቶሙ፡ ነፋስ።</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Malāʾǝktihomu la-śǝllus ruṣatomu nafās</title>
+                <title xml:lang="en" corresp="#t1">The Messenger of the Trinity is the Running Win</title>
```
[the remark this answers is not in the document]
Comment (Ralph-Lee-UK) [reply new/LIT6448HymnTrinity.xml:10]: Agreed

File: new/LIT6446BlessingMary.xml [hunk 7]
```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6446BlessingMary" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ባርክ፡ ወውዳሴ፡ ለማርያም፡ ሥርጉተ፡ ሥላሴ፡</title>
```
[the remark this answers is not in the document]
Comment (Ralph-Lee-UK) [reply new/LIT6446BlessingMary.xml:8]: I have checked with http://www.bl.uk/manuscripts/Viewer.aspx?ref=add_ms_11622_fs001r which is the only image that I can locate of LIT3011RepCh291 and after the opening line the text is quite different. 
Here is the opening of BLadd11622
download
and this is the opening of this hymn
EMIP02402_Weiner367_027

File: new/LIT2382Taamme.xml [hunk 8]
```diff
@@ -0,0 +1,269 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="work" xml:lang="en" xml:id="LIT2382Taamme">
+   <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
```
[the remark this answers is not in the document]
Comment (Ralph-Lee-UK) [reply new/LIT2382Taamme.xml:6]: @thea-m this is my mistake and I'll correct it in the updated pull request.

File: new/LIT6451MMMonkRain.xml (hunk 1 above)
Comment (Ralph-Lee-UK) [reply new/LIT6451MMMonkRain.xml:53]: Added

File: new/LIT6453MalkeaMichaelGabriel.xml (hunk 2 above)
Comment (Ralph-Lee-UK) [reply new/LIT6453MalkeaMichaelGabriel.xml:59]: correct

File: new/LIT1528GadlaZ.xml [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+
+            <div type="edition">
```
[the remark this answers is not in the document]
Comment (Ralph-Lee-UK) [reply new/LIT1528GadlaZ.xml:88]: Done, and this file too edited in its proper location, and removed from 'new'

File: new/LIT6450MMJudgeSabbath.xml [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography"/>
+            <note>Not in <bibl><ptr target="bm:MacomberMiracles"/></bibl></note>
+            <note>Incipit taken from EMIP 2401</note>
+            <div type="edition">
+              <div type="textpart" subtype="incipit">
+                <ab>
+                  ወሀለወት፡ አሐቲ፡ ቤተ፡ ክርስቲያን፡ ዘሕንፅት፡ በስማ፡ ለእግዝእትነ፡… ዘትፌውሰ፡ ሕሙማነ፡ እለ፡ መጽኡ፡
```
[the remark this answers is not in the document]
Comment (Ralph-Lee-UK) [reply new/LIT6450MMJudgeSabbath.xml:59]: correct

**Comment by Ralph-Lee-UK** [comment]
@thea-m I have gone through these now, and have just pushed the edits to the remote branch, so you should see all of the edits now.

**Review by thea-m — APPROVED** [review]
Thanks a lot!

**Pull request merged**